src/ceph-volume/ceph_volume/api/lvm.py

```diff
@@ -1106,14 +1090,21 @@ def get(self, lv_name=None, vg_name=None, lv_path=None, lv_uuid=None, lv_tags=No
         return lvs[0]
 
 
-def create_lv(name_prefix, uuid, vg=None, device=None, extents=None, size=None, tags=None):
+def create_lv(name_prefix,
+              uuid,
+              vg=None,
+              device=None,
+              slots=None,
+              extents=None,
+              size=None,
+              tags=None):
```

Comment:
imo this piece would benefit from type annotations.

Reply:
Agreed but since ceph-volume needs to be py2 compatible, I refrained from adding type annotations.

Reply:
this would look like so:
```suggestion
              tags=None,  # type: Optional[str]
              ):
```

but I'm ok with waiting for py3.

Reply:
Yeah I'm not a huge fan of this crutch. Sad but its probably best to do this once we're on py3. only